### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by ntap-johanneswagner** [NetAppDocs/astra-control-center#100]
Page: [Install Astra Control Center using OpenShift OperatorHub](https://docs.netapp.com/us-en/astra-control-center/get-started/acc_operatorhub_install.html)

we are writing:
kubectl astra packages push-images -m BUNDLE_FILE -r MY_REGISTRY -u MY_REGISTRY_USER -p MY_REGISTRY_PASSWORD

if I'm not completely wrong this should be: 
kubectl-astra packages push-images -m BUNDLE_FILE -r MY_REGISTRY -u MY_REGISTRY_USER -p MY_REGISTRY_PASSWORD

**Comment by netapp-mwallis** [comment]
Hi @ntap-johanneswagner, thanks for bringing this to our attention.  We'll look in to this and get back to you with more information.

**Comment by ntap-johanneswagner** [comment]
@netapp-mwallis we found out that both is working, so it isn't that urgent and more a cosmetic thing.

What we found out during the troubleshooting is, that the way is only working if docker is used at the machine. If podman is used this way doesn't work. In this case the old way (see https://docs.netapp.com/us-en/astra-control-center-2112/get-started/install_acc.html#add-the-images-to-your-local-registry) has to be used. This should definitely be mentioned as right now every customer that wants to deploy has no way to get it working.
